src/library_idbfs.js

```diff
@@ -220,15 +220,15 @@ addToLibrary({
         callback(e);
         return;
       }
-      req.onsuccess = () => { callback(null); };
+      req.onsuccess = callback;
       req.onerror = (e) => {
         callback(e.target.error);
         e.preventDefault();
       };
     },
     removeRemoteEntry: (store, path, callback) => {
       var req = store.delete(path);
-      req.onsuccess = () => { callback(null); };
+      req.onsuccess = callback;
```

Comment:
This doesn't seem right? In other places `onsuccess` gets an `event` argument, which in the previous version was simply ignored and callback was invoked with `null`, whereas now the event will be propagated to the callback which will think it's an error argument instead.

Risks like this is one of the reasons I'm a bit hesitant about changes like this. Brings to my mind this Jake Archibald's post: https://jakearchibald.com/2021/function-callback-risks/

The only safe simplification here seems to be `() => callback()` as that's what some other places are already doing.

Reply:
Ah yes, good point.